### PR TITLE
south-africa_shawen-aldred-harker

### PR DESCRIPTION
**Pull request by ShawenHarker**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: exercises/F-logical-operators/exercise.js [hunk 1]
```diff
@@ -11,14 +11,14 @@ var cssLevel = 4;
...
+var htmlLevelAbove5 = true;
+var cssLevelAbove5 = false;
 
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
-var cssAndHtmlAbove5;
-var cssOrHtmlAbove5;
+var cssAndHtmlAbove5 = htmlLevel > 5 || cssLevel < 5;
+var cssOrHtmlAbove5 = cssLevel > 5 && htmlLevel > 5;
```
Comment (sinopungula) [inline exercises/F-logical-operators/exercise.js:21]: Hi. I am a little confused by your logical operators and how you placed them.

**Review by sinopungula — COMMENTED** [review]
I reviewed your work, great work!

File: exercises/E-conditionals/exercise.js [hunk 2]
```diff
@@ -9,6 +9,10 @@
 var name = "Daniel";
 var danielsRole = "mentor";
 
+if (danielsRole = "mentor") {
+  console.log("Hi, I'm Daniel, I'm a mentor.")
```
Comment (JarrodBen) [inline exercises/E-conditionals/exercise.js:13]: Hey just a heads-up, it's best practice to not hardcode the work, rather replace the words `mentor` and `Daniel` with it's appropriate declared variables.

Comment (JarrodBen) [reply exercises/E-conditionals/exercise.js:13]: Other than that, the rest of your code looks great and I know you've probably had a busy week, but just make sure that the mandatory folder is completed as well as it is due by the end of today.

**Comment by JarrodBen** [comment]
I reviewed your work and the rest of your code looks great and I know you've probably had a busy week, but just make sure that the mandatory folder is completed as well as it is due by the end of today.

**Comment by aashiqabrahams123** [comment]
Hi Shawen, I've reviewed your code, well done on your hard work!!!!

**Pull request closed without merging**